dkg-gadget/src/meta_async_rounds/dkg_gossip_engine/mod.rs

```diff
@@ -53,7 +55,7 @@ pub trait GossipEngineIface: Send + Sync {
 	fn receive_timestamps(&self) -> Option<&ReceiveTimestamp<Self::Clock>>;
 }
 
-pub type ReceiveTimestamp<C> = Arc<RwLock<HashMap<PeerId, (C, Instant, AuthorityId)>>>;
+pub type ReceiveTimestamp<C> = Arc<RwLock<HashMap<PeerId, C>>>;
```

Comment:
Do you think we still need the `ReceiveTimestamp`? we do not report or calculate the misbehaviours on these timestamps anymore, right?

Reply:
We can get rid of this, I see no point in it at the moment since the statemachine's blame will do something similar for us anyways.